datastore/db/models.py

```diff
@@ -418,6 +418,42 @@ class Meta:
             GinIndex(fields=["source_file", "funding_org_ids"]),
         ]
 
+    @classmethod
+    def is_grant_data_acceptable(cls, grant: Dict[str, Any]) -> bool:
```

Comment:
Can we call this something more specific, such as `check_grant_data_tools_compatible` as 'checking acceptable grant data' here is only specific to the tools and potentially a temp fix for it not being integrated into the standard / dqt.

Reply:
Good idea, the name didn't sit right with me, i just couldn't think of a better one.
Also given it's technically standalone from the Grant model, do you think the function is in the right place? I guess it could also be an indepenent function in models.py, or even some seperate utils py or something

Reply:
Yeah I think I'd move the function to the loader module `load_data_package`